### DISCUSSION TITLE
Foreground color isn't what I've configured

### DISCUSSION THREAD

**Discussion by non-trivial0s** [JanDeDobbeleer/oh-my-posh#5530] [Troubleshoot] [answered]
If foreground and background color look both "dark", foreground color will be "lighter". (segment 1)
If foreground and background color look both "bright", foreground color will be "darker". (segment 4)
If one is bright, the other is dark, they display just as configured. (segments 2 & 3)
I know it's to show it more clearly, could you tell me more details?
![image](https://github.com/user-attachments/assets/3edd8f60-ea46-4390-895e-00feacc64ccf)

**Comment by lewis-yeung** [comment]
> I know it's to show it more clearly, could you tell me more details?

@non-trivial0s I think it is beyond the scope of Oh My Posh. We don't have extra tweaks on colors, and we only output ANSI sequences, while the color rendering/display is the responsibility of a terminal program (i.e., Windows Terminal, in this case). Everything looks normal with Windows Terminal v1.20.11381.0 on my end. What's the version of yours?

**Reply by non-trivial0s** [reply, marked as answer]
thank you for your comment💕
you are right, I turned on a setting in 

> Windows terminal -> settings -> defaults -> appearance -> automatically adjust lightness of indistinguishable text

turn it to "never", it looks as configured.
